### PR TITLE
Break 2 Solutions

### PR DESCRIPTION
**Pull request by rajivkr**

### CODE REVIEW

File: 5_6-Jan/alternatingChar.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+let alternatingCharacters = (arrOfInputStrings = []) => {
+  return arrOfInputStrings.map(input => {
+    let minimumDeletionCount = 0;
+    for (const [index, elem] of input.split('').entries()) {
```
Comment (ahhda) [inline 5_6-Jan/alternatingChar.js]: A lot of things happening on a single line, use a variable to store data instead

File: 5_6-Jan/alternatingChar.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+      if (elem === input[index + 1]) minimumDeletionCount++;
```
Comment (ahhda) [inline 5_6-Jan/alternatingChar.js]: Better to let it span multiple lines and have brackets

File: 8_9-Jan/form-attributes.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Pesto Form Attribute</title>
+  </head>
+  <body>
+    <form>
+      First name:
+      <input type="text" name="firstname" value="John" disabled="true" />
+      <input type="submit" />
+    </form>
```
Comment (ahhda) [inline 8_9-Jan/form-attributes.html]: * Label is missing
* Use submit button instead of input tag

File: 8_9-Jan/form.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Pesto Form</title>
+  </head>
+  <body>
+    <form>
+      Quantity (between 1 and 5):
+      <br />
+      <br />
+      <br />
```
Comment (ahhda) [inline 8_9-Jan/form.html]: Not a good idea to use `<br />`. Instead use padding and write a little CSS.

**Comment by rajivkr** [comment]
Thanks @ahhda for taking time to review the code. Will work on that. 
Prettier removes the braces and makes it into single line in those cases.

**Pull request merged**